src/component/gallery/Gallery.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable no-unused-vars */
+import { AnimatePresence, motion } from 'framer-motion';
+import React, { useState } from 'react';
+import { BiLeftArrow } from 'react-icons/bi';
+import useToggle from '../customHooks/useToggle';
+import Avatar from '../plane/Avatar';
+import styles from './gallery.module.scss';
+
+const variants = {
+  initial: (direction) => ({
+    x: direction > 0 ? 500 : -500,
+    opacity: 0,
+  }),
+  animate: {
+    zIndex: 1,
+    x: 0,
+    opacity: 1,
+  },
+  exit: (direction) => ({
+    zIndex: 0,
+    x: direction < 0 ? 500 : -500,
+    opacity: 0,
+  }),
+};
+
+const Gallery = ({ itemList }) => {
+  const [item, setToggle] = useToggle(itemList);
+  const [dir, setDir] = useState(1);
+
+  const handleDir = (dir) => {
+    setToggle(dir);
+    setDir(dir);
+  };
+
+  return (
+    <div className={styles.mainContainer}>
+      <motion.button
+        className={styles.roundedBtn}
+        type="button"
+        onClick={() => handleDir(-1)}
+      >
+        {' '}
+        <BiLeftArrow style={{ color: 'white', fontSize: '1.5rem' }} />
+        {' '}
+      </motion.button>
+      <div className={styles.header}>
+        <h2> Avaliable planes </h2>
+        <p> Please pick a model </p>
+        <p style={{ fontSize: '2rem', fontWeight: 'bold' }}>
```

Comment:
- [ ] If you can use classes for styling here, it would be nice, since it's not a good practice to use the `style` attribute.